lib/modules/platform/bitbucket-server/utils.ts

```diff
@@ -11,6 +12,7 @@ import type {
   HttpResponse,
 } from '../../../util/http/types';
 import { getPrBodyStruct } from '../pr-body';
```

Comment:
```suggestion
import { ensureTrailingSlash, parseUrl } from '../../../util/url';
import { getPrBodyStruct } from '../pr-body';
```

Reply:
`ensureTrailingSlash` doesn't appear to be used, is it missing from another change?

Reply:
yes, sorry, leftover from testing